hyp3_autorift/vend/testautoRIFT.py

```diff
@@ -981,7 +985,7 @@ def generateAutoriftProduct(indir_m, indir_s, grid_location, init_offset, search
                         parameter_file=kwargs['parameter_file'],
                     )
 
-                elif nc_sensor == "L7":
+                elif (nc_sensor == "L7") | (nc_sensor == "L5") | (nc_sensor == "L4"):
```

Comment:
```suggestion
                elif nc_sensor in ["L7", "L5", "L4"]:
```
I think you can simplify this to

Reply:
Made this change!